homeassistant/components/synology_dsm/services.yaml

```
@@ -0,0 +1,15 @@
+# synology-dsm service entries description.
+
+reboot:
+  description: Reboots the NAS.
+  fields:
+    serial:
+      description: serial of NAS to reboot; optional when only one is configured.
+      example: 1NDVC86409
+
+shutdown:
+  description: Shutdown the NAS.
+  fields:
+    serial:
+      description: serial of NAS to shutdown; optional when only one is configured.
```

```suggestion
      description: serial of the NAS to shutdown; required when multiple NAS are configured.
```

I see it the other way around. What do you think of that ?

---

your suggestion sounds much more convenient ... will also update the doc PR